src/qt/forms/sendcoinsdialog.ui

```diff
@@ -1108,10 +1108,10 @@
            <item>
             <widget class="QCheckBox" name="optInRBF">
              <property name="text">
-              <string>Request Replace-By-Fee</string>
+              <string>Allow Fee Increase</string>
              </property>
              <property name="toolTip">
-              <string>Indicates that the sender may wish to replace this transaction with a new one paying higher fees (prior to being confirmed).</string>
+              <string>If the transaction takes a long time to confirm, this allows you to increase the fee later ("Replace-By-Fee", BIP 125). The recommended fee will be lower.</string>
```

Comment:
maybe shorten it to "If the transaction takes too long to confirm" ?

Reply:
This is one of the possible use cases. How about *cancelling*? And there is also the confirmations detail that concerns the recipient. There are too many details in the BIP to make this copy simple.

Reply:
I'm more fond of the previous tooltip as it doesn't mention confirmation time. Maybe just add BIP 125 reference?

Reply:
> I'm more fond of the previous tooltip as it doesn't mention confirmation time

Agree, and would at least shift the emphasis away if this needs to be kept, changing from "If the transaction takes a long time to confirm, this allows you to increase the fee later" to "This allows you to increase the fee later if the transaction takes a long time to confirm."

It might also be clearer to change "The recommended fee will be lower" to ""This will cause the recommended fee to be lower."